testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/components/Commons/ChildrenEditor.java

```diff
@@ -67,6 +67,7 @@ public void removeFirstItem() {
      * @throws InterruptedException
      */
     public void moveItems(int dragElement,int targetElement) throws InterruptedException {
+        Commons.webDriverWait(1000);
```

Comment:
I'd rather we define a constant for this.

Reply:
Yes, I'll update that.